culture/review_culture.md

```diff
@@ -0,0 +1,142 @@
+> If you want to go fast, go alone. If you want to go far, go together.
+
+As a community, we value going far over going fast. To go far, we need to stay
+aligned in our values, goals, and practices. One of the ways that we do this is
+through cultivating review culture.
+
+# On Review Culture
+
+Review culture came to us from the practices of software engineers. Large
+codebases are incredibly complex: they contain the ideas and work of hundreds
+of people, manifested in millions of lines of code. They need to remain
+logically consistent, even as they grow and change. A big part of how engineers
+manage is through code review: every single change to the system is reviewed by
+at least one other person before becoming official, or "merging".
```

Comment:
Counter-suggest keeping it as is. Non-developers are an important part
of the audience of this document, if not the primary audience. Nowhere
else in this guide is the term `master` mentioned, nor need it be. We
should strive to be technically accurate to the extent that we do not
provide misleading information, but minute precision muddles the forest
for the trees.

Reply:
Agreed with @wchargin -- non-developers are a core intended audience for this document, so I want to include technical lingo only when it's high lift. I think "merged" just passes that test, "merged to master" is too much detail.

Reply:
Sure that's fair. GitHub's UI does promote the merge very prominently. And on our GitHub you won't see merges into other branches very often.

![image](https://user-images.githubusercontent.com/497556/72337064-2de8ac80-36c2-11ea-9092-33d3e00d8505.png)

My thinking was that the master was more important than the motion of merging. Because if something ends up in master by other means (like directly pushing by the power of maintainership) it still has the official stamp on it. And this sentence is about "what makes something official".

So I have a preference to explain "master" over "merging". But since this document explains the reviewing workflow, you can't avoid "merging". Hence merging into master.

It's difficult for me to say what is needed here though. I'm not exactly the intended audience and haven't tried to explain git to beginners recently.

Happy to leave this as-is and amend later if it needs more details.